WcaOnRails/config/application.rb

```diff
@@ -1,5 +1,6 @@
 require File.expand_path('../boot', __FILE__)
-require File.expand_path('../../lib/fix_accept_header', __FILE__)
+require_relative '../lib/middlewares/fix_accept_header'
```

Comment:
The first line was auto-generated by Rails, so if we want to update Rails, this shouldn't lead to a conflict.
To be honest I'm not really sure why they don't use `require_relative` there. They use it in `bin/rake` though.